kafka-ui-api/src/test/java/com/provectus/kafka/ui/KafkaTopicCreateTests.java

```diff
@@ -57,4 +57,25 @@ void shouldReturn400IfTopicAlreadyExists() {
         .expectStatus()
         .isBadRequest();
   }
+
+  @Test
+  void shouldRecreateExistingTopicSuccessfully() {
+    TopicCreationDTO topicCreation = new TopicCreationDTO()
+            .replicationFactor(1)
+            .partitions(3)
+            .name(UUID.randomUUID().toString());
+
+    webTestClient.post()
+            .uri("/api/clusters/{clusterName}/topics", LOCAL)
+            .bodyValue(topicCreation)
+            .exchange()
+            .expectStatus()
+            .isOk();
+
+    webTestClient.post()
+            .uri("/api/clusters/{clusterName}/topics/" + topicCreation.getName(), LOCAL)
+            .exchange()
+            .expectStatus()
```

Comment:
according to swagger spec status should be 201

Reply:
also, please check that newly create topic setting(like partitions count are the same )

Reply:
Fixed.